src/app/options.cpp

```diff
@@ -247,6 +252,12 @@ namespace
     constexpr const StringOption CONFIGURATION_OPTION = {"configuration"};
     constexpr const BoolOption PORTABLE_OPTION = {"portable"};
     constexpr const BoolOption RELATIVE_FASTRESUME = {"relative-fastresume"};
+    constexpr const StringOption SAVE_PATH_OPTION = {"path"};
+    constexpr const StringOption STATUS_OPTION = {"status"};
```

Comment:
@glassez Yes, that could work. I originally discounted that as BoolOption doesn't current allow a way to specify a value, and I don't think it should since it's implementing a specific common kind of command line option. Instead I could continue using StringOption and checking its value, requiring it to be true or false. That would be the easiest and fastest change.

Alternatively, I could add a new Option-derived class called something like TriStateBoolOption, that requires explicitly specifying a value. That would allow some flexibility in how the value is specified, i.e. true, false, 1, 0, on, and off could all be valid values.